docs/source/usrman/tutorial.rst

```diff
@@ -102,7 +105,7 @@ Point-to-Point Communication
    from mpi4py import MPI
 
    comm = MPI.COMM_WORLD
-   rank = comm.Get_rank()
+   rank = comm.Get_rank
```

Comment:
I don't understand, `Get_rank` etc are now attributes instead of callables?

Reply:
Oh, no! I messed things up with brute-force search & replace.

Reply:
Fixed.